### ISSUE TITLE
After training, images of outputs are black! Only black backgroud. What should I do 

### ISSUE THREAD

**Issue by xhw205** [DrSleep/tensorflow-deeplab-resnet#153]

**Comment by VanitarNordic** [comment]
mostly there is something wrong with your training images, annotations

**Comment by xhw205** [comment]
Thanks，I have solved the problem.The reason is the training image must single channel，not RGB。

**Comment by stiffler123** [comment]
do you mean train image are  single channel picture？？ or the mask（annotion） must be single channel

**Comment by xhw205** [comment]
Yeah.e.g. Voc2012Aug...Just segmentation's ground truth use single channel (The conversion may be done by yourself )，and training images are still RGB.

**Comment by stiffler123** [comment]
ok，today i  already try to  convert rgb label(0,0,128) to single channel (1) . train project will be done tomorrow  . hope it may work.

thanks for you help.





--
此致敬礼！

At 2018-01-03 18:42:11, "xhw205" <notifications@github.com> wrote:


Yeah.e.g. Voc2012Aug...Just segmentation's ground truth use single channel (The conversion may be done by yourself )，and training images are still RGB.

**Comment by xhw205** [comment]
Good luck！

**Comment by SehgalDivij** [comment]
I have to detect boats. I have a images and binary masks for images for the object of interest. How do I label the input? As in, I have the mask and image to which it belongs but a label for the object represented is missing. If I want to train on this, how does the model figure out where the object is in the image and where is the mask for the target object and what kind of object the mask represents(person/cat/dog/car/table/chair/boat).

Currently, my mask is rendered as in the second part of the following image:
[image + mask + mask superimposed on image](https://files.slack.com/files-pri/TAQUC9H99-FCR2TCCQ7/image.png)

**Comment by apolo74** [comment]
Hi guys, I'm using single channel annotations for training but I'm still getting black images in the output after training... Are you sure that is the problem? Any other ideas?
